### ISSUE TITLE
Image/MauiImage does not render a particular SVG

### ISSUE THREAD

**Issue by janusw** [dotnet/maui#19448]
### Description

I noticed that the following SVG file is not being rendered properly, when included as a MauiImage and referenced in an `Image` as png:

https://fonts.gstatic.com/s/i/short-term/release/materialsymbolsoutlined/directions_boat/default/48px.svg

```
<Image Source="directions_boat.png" />
```

All other SVG files that I tried work without a problem.

### Steps to Reproduce

1. Clone the repo below (a fork of dotnet/maui-samples).
2. Check out the branch "MauiImage_svg_bug_repro"
3. Open 8.0/UserInterface/ControlGallery/ControlGallery.sln
4. Run it on an Android or iOS device.
5. Select "Image (Display a bitmap)"
6. Observe that the car is rendered, but the boat is not.

### Link to public reproduction project repository

https://github.com/janusw/maui-samples/tree/MauiImage_svg_bug_repro

### Version with bug

8.0.3

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, Android

**Comment by PureWeen** [comment]
@janusw you'll need to change the extension to ".png" for it to render the image

```XAML
<Image Source="directions_boat.png" />
```

**Comment by janusw** [comment]
> @janusw you'll need to change the extension to ".png" for it to render the image
> 
> ```
> <Image Source="directions_boat.png" />
> ```

Ah, yes, you're right. On iOS this will only work with png (I was primarily testing on Android, where in fact also svg works).

BUT: The boat icon still does not show up when using png (only the car does). There must be something specific about this svg that makes it fail.

**Comment by janusw** [comment]
I'm attaching the generated pngs from my build directory (obj/Debug/net8.0-ios/ios-arm64/resizetizer/r/). They're empty:

[directions_boat.png](https://github.com/dotnet/maui/assets/484108/829c5e25-b0e2-4856-94c0-286dfc1c9ff8)
[directions_boat@2x.png](https://github.com/dotnet/maui/assets/484108/6f83fd97-7812-4f89-a3c4-356cc1d020d3)
[directions_boat@3x.png](https://github.com/dotnet/maui/assets/484108/80710eac-fb32-4330-8640-8ad4a3431278)

**Comment by janusw** [comment]
possibly related to https://github.com/Redth/ResizetizerNT/issues/43 ?

**Comment by PureWeen** [comment]
@mattleibow thoughts?

**Comment by jkommeren** [comment]
Im having a possibly related issue where my svg is not rendered when used as the app icon, when shown as share destination in Android.

I'll create a ticket for it with demo project. The image worked in .net 7.

**Comment by janusw** [comment]
> Im having a possibly related issue where my svg is not rendered when used as the app icon, when shown as share destination in Android.
> 
> I'll create a ticket for it with demo project. The image worked in .net 7.

This case must be slightly different than mine (mine fails already with .NET 7).

**Comment by jkommeren** [comment]
> > Im having a possibly related issue where my svg is not rendered when used as the app icon, when shown as share destination in Android.
> > 
> > I'll create a ticket for it with demo project. The image worked in .net 7.
> 
> This case must be slightly different than mine (mine fails already with .NET 7).

Ok, I'll mention this. Thanks!

**Comment by janusw** [comment]
The problem was still occurring up to version 8.0.14, but luckily it seems like it has been fixed in version 8.0.20, most probably via #16100. 🥳
